python/lsst/ts/wep/task/calcZernikesNeuralTask.py

```diff
@@ -0,0 +1,250 @@
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = [
+    "CalcZernikesNeuralTaskConnections",
+    "CalcZernikesNeuralTaskConfig",
+    "CalcZernikesNeuralTask",
+]
+
+from typing import Any, Optional
+import torch
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+import numpy as np
+from astropy.table import QTable
+from lsst.pipe.base import connectionTypes
+from lsst.utils.timer import timeMethod
+from TARTS import NeuralActiveOpticsSys
+
+class CalcZernikesNeuralTaskConnections(
+    pipeBase.PipelineTaskConnections,
+    dimensions=("visit", "detector", "instrument"),  # type: ignore
+):
+
+    donutStampsExtra = connectionTypes.Input(
+        doc="Extra-focal Donut Postage Stamp Images",
+        dimensions=("visit", "detector", "instrument"),
+        storageClass="StampsBase",
+        name="donutStampsExtra",
+    )
+    donutStampsIntra = connectionTypes.Input(
+        doc="Intra-focal Donut Postage Stamp Images",
+        dimensions=("visit", "detector", "instrument"),
+        storageClass="StampsBase",
+        name="donutStampsIntra",
+    )
+    outputZernikesRaw = connectionTypes.Output(
+        doc="Zernike Coefficients from all donuts",
+        dimensions=("visit", "detector", "instrument"),
+        storageClass="NumpyArray",
+        name="zernikeEstimateRaw",
+    )
+    outputZernikesAvg = connectionTypes.Output(
+        doc="Zernike Coefficients averaged over donuts",
+        dimensions=("visit", "detector", "instrument"),
+        storageClass="NumpyArray",
+        name="zernikeEstimateAvg",
+    )
+    zernikes = connectionTypes.Output(
+        doc="Zernike Coefficients for individual donuts and average over donuts",
+        dimensions=("visit", "detector", "instrument"),
+        storageClass="AstropyQTable",
+        name="zernikes",
+    )
+    donutQualityTable = connectionTypes.Output(
+        doc="Quality information for donuts",
+        dimensions=("visit", "detector", "instrument"),
+        storageClass="AstropyQTable",
+        name="donutQualityTable",
+    )
+
+
+class CalcZernikesNeuralTaskConfig(
+    pipeBase.PipelineTaskConfig,
+    pipelineConnections=CalcZernikesNeuralTaskConnections,  # type: ignore
+):
+    """Configuration for CalcZernikesNeuralTask.
+
+    Attributes
+    ----------
+    wavenet_path : str
+        Model Weights Path for wavenet
+    alignet_path : str
+        Model Weights Path for alignet
+    aggregatornet_path : str
+        Model Weights Path for aggregatornet
+    dataset_param_path : str
+        datasetparam path
+    device : str
+        Device to use for calculations
+    """
+
+    wavenet_path = pexConfig.Field(  # type: ignore
+        doc="Model Weights Path for wavenet",
+        dtype=str
+    )
+    alignet_path = pexConfig.Field(  # type: ignore
+        doc="Model Weights Path for alignet",
+        dtype=str
+    )
+    aggregatornet_path = pexConfig.Field(  # type: ignore
```

Comment:
I don't think any of this type: ignore should be needed. At least judging by all the other tasks in ts_wep, I couldn't see this being used there, why would it be needed here?